### PR TITLE
Update lan_admin.php

### PR DESCRIPTION
**Pull request by yesszus**

### CODE REVIEW

**Comment by yesszus** [comment]
@CaMer0n  pls review this.

File: e107_handlers/form_handler.php [hunk 1]
```diff
@@ -1701,7 +1701,7 @@ function file($name, $options = array())
 
 	function upload($name, $options = array())
 	{
-		return 'Ready to use upload form fields, optional - file list view';
+		return LAN_READY_UP_FOLDER_FIELDS;
```
Comment (CaMer0n) [inline e107_handlers/form_handler.php]: This is debug info

File: e107_handlers/form_handler.php [hunk 2]
```diff
@@ -906,15 +906,15 @@ private function mediaUrl($category = '', $label = '', $tagid='', $extras=null)
 		if(E107_DBG_BASIC)
 		{
 
-			$title = "Media Manager : ".$category;
+			$title = LAN_MEDIAMANAGER." : ".$category;
```
Comment (CaMer0n) [inline e107_handlers/form_handler.php]: this is debug info

**Comment by yesszus** [comment]
Restored.

**Comment by yesszus** [comment]
New form_handler.php hardcoded progress?

**Comment by CaMer0n** [comment]
Yes, the form_handler is used everywhere inside e107. So using a LAN from `lan_image.php` or `lan_user.php` or even `lan_admin.php` inside the form_handler is a **bad** idea.  The form_handler should contain only generic LANs from `e107_languages/English/English.php` and terms from `lan_form_handler.php`.  Please add terms to lan_form_handler.php as you feel is necessary. Thank you!

@MikeyGMT fyi also.

**Pull request closed without merging**